modules/microshift-checking-pods-mismatch.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-checking-pods-mismatch_{context}"]
+= Checking the pods for mismatch
+
+Check if the Pods have a mismatch using the following procedure
+
+.Procedure
+
+. Check the context and permissions inside the first pod by running the following command:
++
+[source,terminal]
+----
+$ oc rsh <pod name A> ls -lZah /<PVC mountpoint>
```

Comment:
How do I determine my PVC mountpoint ?

Reply:
To just list the mount points within a pod, users could use

`oc get pods -n my-ns my-pod -ojsonpath='{.spec.containers[*].volumeMounts[*].mountPath}'
`

Or else users should be directed to the output of `oc describe pod ...` for the pod's mountpoints.

Reply:
@copejon  and @dhensel-rh I have added the command as suggested. It does not have the output example. 
Should this be added? Please check for any changes to be made?

Reply:
see suggested change above

Reply:
done. Please check @copejon and @dhensel-rh